src/Language/PureScript/Errors.hs

```diff
@@ -556,6 +558,16 @@ prettyPrintSingleError full e = prettyPrintErrorMessage <$> onTypesInErrorMessag
             ]
     goSimple (WildcardInferredType ty) =
       line $ "The wildcard type definition has the inferred type " ++ prettyPrintType ty
+    goSimple (NotExhaustivePattern bs) =
+      paras $ [ line "Pattern Match(es) are not exhaustive"
```

Comment:
Purely cosmetic: maybe `Pattern match is not exhaustive`? I tend to think of the whole `case` as a singular "pattern match" (although I might be the only one). In any case, I think it makes the error slightly easier to read.

Also, I think a line break might be nice between the second line and the list of cases.